docs/pages/build-reference/apk.mdx

```diff
@@ -35,7 +36,7 @@ By default, EAS Build produces Android App Bundle, you can change it by modifyin
 }
 ```
 
-Now, to run your build run `eas build -p android --profile preview`. Remember that you can name the profile whatever you like; we named the profile "preview", but you could call it "local" or "simulator", whatever makes most sense for you.
+Now, to run your build run `eas build -p android --profile preview1`. Remember that you can name the profile whatever you like; we named the profile `preview1`, but you could call it `local`, `simulator` or whatever makes most sense for you.
```

Comment:
```suggestion
Now you can run your build with `eas build -p android --profile preview`. Remember that you can name the profile whatever you like. We named the profile `"preview"`, but you can call it `"local"`, `"simulator"`, or whatever makes the most sense for you.
```

For clarity, let's also stick to the "preview" profile convention here.

Unrelated to this PR: Some grammatical suggestions and improving the flow of the first sentence.

Reply:
Agree with keeping `preview` over `preview1`.

Are you sure you want both ` and " around the label names? Looks a bit weird.

Reply:
We genereally highlight the properties with inline code syntax. For example, see `runtimeVersion` property [here](https://docs.expo.dev/build-reference/app-versions/#limitations). I think this will be a nice improvement.

Reply:
Cool